.buildkite/lm-eval-harness/run-lm-eval-gsm-vllm-baseline.sh

```diff
@@ -46,6 +46,6 @@ while getopts "m:b:l:f:t:" OPT; do
 done
 
 lm_eval --model vllm \
-  --model_args pretrained=$MODEL,tensor_parallel_size=$TP_SIZE \
+  --model_args pretrained=$MODEL,tensor_parallel_size=$TP_SIZE,distributed_executor_backend="ray" \
```

Comment:
I thought ray was used by default if it was installed, does it need to be specified manually now?

Reply:
`mp` is default

Was having issues using this inside lm-eval-harness with `mp` unrelated to this PR